### PR TITLE
Add 88f6282 to archs needing mono workaround

### PR DESCRIPTION
**Pull request by Stanzilla**
_Motivation:_  Explain here what the reason for the pull request is.
_Linked issues:_  Optionally, add links to existing issues or other PR's

### Checklist
- [ ] Build rule `all-supported` completed successfully
- [ ] Package upgrade completed successfully
- [ ] New installation of package completed successfully

### CODE REVIEW

File: spk/sonarr/src/service-setup.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ fi
 LEGACY_CONFIG_DIR="${SYNOPKG_PKGDEST}/.config"
 
 # workaround for mono bug with armv5 (https://github.com/mono/mono/issues/12537)
-if [ "$SYNOPKG_DSM_ARCH" == "88f8621" -o "$SYNOPKG_DSM_ARCH" == "88f8622" ]; then
+if [ "$SYNOPKG_DSM_ARCH" == "88f8621" -o "$SYNOPKG_DSM_ARCH" == "88f8622" -o "$SYNOPKG_DSM_ARCH" == "88f6282" ]; then
```
Comment (hgy59) [inline spk/sonarr/src/service-setup.sh:33]: The correct fix would be  `[ "$SYNOPKG_DSM_ARCH" == "88f6281" -o "$SYNOPKG_DSM_ARCH" == "88f6282" ]`
This is for `88f628x` (synology) and `88f6281` (SynoCommunity) arches for models like DS213air, DS213, DS413j, DS112, DS112+, DS212, DS212+, RS212, RS812, DS212j, DS112j, and more.
"88f8621" and "88f8622" are invalid values.

**Review by hgy59 — CHANGES_REQUESTED** [review]

File: spk/sonarr/src/service-setup.sh (hunk 1 above)
Comment (Stanzilla) [reply spk/sonarr/src/service-setup.sh:33]: ah so it is currently wrong as well?

**Comment by ymartin59** [comment]
Mono may also require a complete update but previous maintainer no longer runs applications on DSM, so I have no idea who may invest time for such a hard work.
It is expected that applicative packages move to dotNetCore compilation

**Review by hgy59 — APPROVED** [review]

**Comment by hgy59** [comment]
@Stanzilla this fix is now published to the synocommunity package repository. You should see this within 48 h in the DSM package center.

**Pull request merged**